package.json

```diff
@@ -36,7 +36,7 @@
     "reporter-options": "configFile=.mocha-multi.json"
   },
   "dependencies": {
-    "@adobe/lit-mobx": "2.2.2",
```

Comment:
Removed `@adobe/lit-mobx` since we are not using it and I don't think it adds to much